soapclient/SforceEnterpriseClient.php

```diff
@@ -50,7 +47,8 @@ function SforceEnterpriseClient() {
    * @return SaveResult
    */
   public function create($sObjects, $type) {
-    foreach ($sObjects as &$sObject) {
+    $arg = [];

```

Comment:
By using the short array syntax, your are making the code not compatible with version older than PHP 5.4

https://wiki.php.net/rfc/shortsyntaxforarrays

So it would break on PHP 5.3

Everyone will agree that people are not supposed to use a PHP version that is no longer supported, but some people are glued on old version.

As the code change to make it compatible with older version, is very simple, I would like to recommend you to use the "classic" array definition.

$arg = array();